index.html

```diff
@@ -8,9 +8,80 @@
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.9/css/all.css" integrity="sha384-5SOiIsAziJl6AWe0HWRKTXlfcSHKmYV4RBF18PPJ173Kzn7jzMyFuTtk8JA7QQG1" crossorigin="anonymous">
+
 </head>
 <body>
 
+        <nav class="navigation">
+     <div class="content">
+       <ul class="links">
+         <li class="active">Meet Karma</li>
+         <li>How it Works</li>
+         <li>Store</li>
+         <li>Blog</li>
+         <li>Help</li>
+         <li>Login</li>
+       </ul>
+       <img class="logo" src="img/karma-logo.svg"  width="26">
+     </div>
+   </nav>
+   <section class="intro">
+     <div class="content">
+       <h1>Introducting Karma</h1>
+       <h3>Bring WiFi with you, everywhere you go.</h3>
+       <div class="button">
+         <button>Learn More</button>
+       </div>
+     </div>
+   </section>
+   <section class="cases">
+     <div class="content">
+       <h2>Everyone needs a little Karma.</h2>
```

Comment:
Again, try adding `display: flex` to the `.content` container and position the list items that way.
To avoid the devices spreading too wide, contain them with a `max-width`